code/datums/elements/ELEMENT_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+
+# Template file for your new element
+
+See _element.dm for detailed explanations
+
+```dm
+/datum/element/myelement
+	element_flags = ELEMENT_BESPOKE | ELEMENT_COMPLEX_DETACH | ELEMENT_DETACH | ELEMENT_NOTAREALFLAG    // code/__DEFINES/dcs/flags.dm
+	//id_arg_index = 2                                                                                  // Use with ELEMENT_BESPOKE
+	var/list/myvar = list()
+
+/datum/element/myelement/Attach(datum/target)
+	myvar |= target
```

Comment:
This sort of code should not be in the template as it's rare that an element should be keeping track of state. Having it in the template implies that it is generally acceptable which it isn't.

Reply:
I want some kind of example code here, however tiny. Maybe just a simple `target.name` change.

Reply:
Show something like playing a sound on move, just changing a var isn't really a good use of an element either.